src/common/config.cpp

```diff
@@ -821,6 +831,7 @@ void Config::GenData() {
   ul_encoded_bits_.Free();
   dl_mod_input_.Free();
   FreeBuffer1d(&pilot_ifft);
+  std::free(scramble_buffer_ptr);
```

Comment:
See above comments.   Change to delete if you feel this is a good suggestion, ignore otherwise